go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/firecracker-microvm/firectl
 
 require (
-	github.com/firecracker-microvm/firecracker-go-sdk v0.0.0-20190226020201-847e85db1412
+	github.com/firecracker-microvm/firecracker-go-sdk v0.15.1-0.20190301211204-479ac1377a66
```

Comment:
Where did this get 0.15.1?

Reply:
I think `go mod` made it up. Since 1.15.0 exists as a tag, and the commit in question is newer than that, it chose 1.15.1.

Reply:
Are you intending to use a commit newer than 0.15.0 or was that unintentional?

Reply:
Unfortunately, even though firecracker 0.15.0 adds the SendCtrlAltDel API, we didn't add the machine.Shutdown() method in the SDK 0.15.0 release. IMO this oversight is a bug, and I'd not be opposed to releasing SDK 0.15.1 to fix it, but that's a separate matter.

Reply:
Want to open an issue for an 0.15.1 that includes the machine.Shutdown() method?  I'm okay with what you're doing right now (i.e., I don't want to block this merge) but we should get the method in an actual release.

Reply:
This is firecracker-microvm/firecracker-go-sdk#82

Reply:
Go SDK 0.15.1 is now available, including support for machine.Shutdown(). I've updated go.mod and go.sum in this PR to use that release.